packages/strapi-plugin-content-manager/admin/src/components/DynamicZone/CategoryItem.js

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Collapse } from 'reactstrap';
+import Banner from './Banner';
+import DynamicComponentCard from '../DynamicComponentCard';
+
+const CategoryItem = ({
+  category,
+  components,
+  isOpen,
+  isFirst,
+  onClickToggle,
+  onClickComponent,
+}) => {
+  const [showComponents, setShowComponents] = useState(false);
+  useEffect(() => {
+    if (isOpen) {
+      setShowComponents(true);
+    }
+  }, [isOpen]);
+
+  return (
+    <>
+      <Banner onClickToggle={onClickToggle} isFirst={isFirst} isOpen={isOpen} category={category} />
+      <Collapse
+        isOpen={isOpen}
+        style={{ backgroundColor: '#F3F3F3' }}
+        onExited={() => setShowComponents(false)}
```

Comment:
Can you create a handle function please.


```js
const handleExited = () => setShowComponents(false);

return (
    <>
      <Banner onClickToggle={onClickToggle} isFirst={isFirst} isOpen={isOpen} category={category} />
      <Collapse
        isOpen={isOpen}
        style={{ backgroundColor: '#F3F3F3' }}
        onExited={handleExited}
```

Reply:
@HichamELBSI done in 146bd85c3058d8ce6fea9a28585569004591faf1